### ISSUE TITLE
First configuration

### ISSUE THREAD

**Issue by gamer06** [0x2142/frigate-notify#174]
Hello everyone, I discovered this notifications feature today. And I was asking for your help, I can't seem to get it started. The path of the ym file. is incorrect, but isn't that how it should be?

![image](https://github.com/user-attachments/assets/86bd9e93-c9e9-42aa-ba9a-b47f6bd3d4cf)
![image](https://github.com/user-attachments/assets/8e79dc29-6c56-4e5d-bb0f-314991dd98cf)

``

**Comment by 0x2142** [comment]
Hi there, 
    Please check the example in the [docs](https://frigate-notify.0x2142.com/latest/install/#docker-recommended), the correct mount within the app is `:/app/config.yml`

**Comment by gamer06** [comment]
Yes, I tried too, but it doesn't work... the location is right, I don't know why it's not finding the config.


![image](https://github.com/user-attachments/assets/f8a1a227-21f3-4003-a0bf-8593e4183b50)
![image](https://github.com/user-attachments/assets/5d1a19a6-3237-41d5-9cae-1572dd0a3054)

**Comment by 0x2142** [comment]
Ah, okay I see - I also notice that your local config is mapped to `/frigatenotify/config.yml`. That's going to point to a root directory at `/frigatenotify`. My guess is you want to change that to `./frigatenotify/config.yml` to point to the current directory.

**Comment by gamer06** [comment]
thanks.  I can already get it to work.  But I realized the following, if I want to open a PIC or clip outside the house, it won't work?  because the IP address is the address of the frigate on my internal network at home (http)...

**Comment by freefd** [comment]
@gamer06, please close this issue if it's no longer relevant.

**Comment by 0x2142** [comment]
Closing after no response
